hooks/prepare-commit-msg

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+#
+# Automatically adds story id to every commit message.
+#
+MAX_MSG_LENGHT=72
+IS_A_NUMBER_REGEX='^[0-9]+$'
+
+BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+SPLITTED_BRANCH=($(echo "$BRANCH_NAME" | tr '-' '\n'))
+STORY_ID="${SPLITTED_BRANCH[0]}"
+
+COMMIT_MSG=$(cat ${1})
+
+TOTAL_MSG_LENGHT=$((${#COMMIT_MSG}+${#STORY_ID}+3))
+
+if (( $TOTAL_MSG_LENGHT < $MAX_MSG_LENGHT)); then
+   if [[ $STORY_ID =~ $IS_A_NUMBER_REGEX ]] ; then
+      sed -i '.bak' "1s/^/[#$STORY_ID] /" $1
+   fi
+else
+   echo "Commit message longer than ${MAX_MSG_LENGHT} characters"
+   exit 1
+fi
```

Comment:
Should we invert the check? This way we apply no changes if we found no story point (it's the case we have no constrained logic on messages).

```suggestion
if [[ $STORY_ID =~ $IS_A_NUMBER_REGEX ]] ; then
   if (( $TOTAL_MSG_LENGHT < $MAX_MSG_LENGHT)); then
      sed -i '.bak' "1s/^/[#$STORY_ID] /" $1
   else
      echo "Commit message longer than ${MAX_MSG_LENGHT} characters"
      exit 1
   fi
fi 
```

Reply:
Yes, you're right! Fixed!